package.json

```diff
@@ -29,7 +29,7 @@
     "tslint": "tslint",
     "sync-colors": "cd externals/elan && git fetch && git checkout origin/master && cp components/lib/variables/colors.json ../../data",
     "sync-externals": "npm run-script sync-schema && npm run-script sync-colors",
-    "sync-schema": "cd externals/metaphysics && git fetch && git checkout origin/master && yarn install && npm run dump-schema -- ../../data && rm -rf node_modules",
+    "sync-schema": "cd externals/metaphysics && git fetch && git checkout origin/message_details_from_impulse && yarn install && npm run dump-schema -- ../../data && rm -rf node_modules",
```

Comment:
@mzikherman small tip: 

You can run the following command locally to sync the schema of a metaphysics branch you're currently working on

`cd <metaphysics_root_dir> && npm run dump-schema -- <path_to_emission_root>/data`